code/2.arbeit.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Arbeitserfahrung</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+<body>
+    
+    <div class="menu">
+        <menu>
+          <a href="./index.html">Home</a>  
+          <a href="./2.arbeit.html">Arbeitserfahrung</a>
+          <a href="./3.ausbildung.html">Ausbildung</a>
+          <a href="./4.sprachen.html">Sprachen</a>
+          <a href="./5.weiteres.html">Weiteres</a>
+          <a href="./6.IT.html">IT Kentnisse</a>
+          <a href="./7.anhang.html">Anhang</a>
+        </menu>
+    </div>
+    
+ <h1>Arbeitserfahrung</h1>   
+
+ <section class="arbeit">  
+
+    <div class="mom">
+      <div class="arbetinfo">
+        <h4>Jan 2020 - heute</h4>
+        <h2>Content Manager</h2>
+        <h3>perfecthair.ch, Wallisellen</h3>
+      </div>
+        <div class="kid">
+        <p>orem ipsum dolor sit amet, consectetur adipiscing elit. Donec vestibulum mi magna, sit amet sollicitudin felis fermentum eget. Sed faucibus urna vitae dolor hendrerit sodales. Nam at aliquet libero, eget commodo elit. Integer dapibus mi nibh, ut euismod magna finibus non. Curabitur laoreet pulvinar mauris. Vestibulum consectetur sed ligula ut iaculis. Sed vel auctor lacus, eu venenatis elit..</p>
+        </div>
+    </div>

```

Comment:
A suggestion would be to give these divs more descriptive class names. The first one could perhaps be called `container` and the kid could be called `text-container`?